projects/manage-supervision-and-delius/README.md

```diff
@@ -1,3 +1,45 @@
-# manage-supervision-and-delius
+# Manage People on Probation (MPoP) and Delius
```

Comment:
The integration service is still called 'Manage Supervision and Delius'. We took the decision to leave it as-is for the moment